packages/squiggle-lang/src/value/index.ts

```diff
@@ -135,17 +178,46 @@ export type LabeledDistribution = {
   distribution: BaseDist;
 };
 
-export type Plot = {
-  distributions: LabeledDistribution[];
-};
-
-class VPlot {
+export type Plot =
+  | {
+      type: "distributions";
+      distributions: LabeledDistribution[];
+    }
+  | {
+      type: "fn";
+      fn: Lambda;
+      min: number;
+      max: number;
+    };
+
+class VPlot implements Indexable {
```

Comment:
I realize this was there before, but ``VPlot`` seems like a bad name, little clue what it means looking at this PR

Reply:
Do you dislike the "Plot" name or "V" prefix? The prefix is common for all value objects, and since they're so common I don't want to make it longer; "PlotValue" would be worse, I think.

Reply:
Ah, didn't realize there was a convention. That's fine then.